src/dex-ui/pages/ProposalDetails/ProposalDetailsControls.tsx

```diff
@@ -34,9 +36,18 @@ export function ProposalDetailsControls(props: ProposalDetailsControlsProps) {
     areButtonsHidden,
     isHasVotedMessageVisible,
     isExecuteButtonVisible,
+    isWalletConnected,
+    doesUserHaveGodTokens,
   } = props.proposalDetails;
 
-  const isUserProposalCreator = proposal.data?.author === wallet.getSigner().getAccountId().toString();
+  let isUserProposalCreator = false;
```

Comment:
Note for the future - we can create a shareable and abstracted `wallet.isCreator` function to determine if the current user is the creator of a contract object.

Reply:
Noted, may be I forgot to mention I have put a `try-catch` block here because if the wallet is not connected this line was failing rendering a complete white screen. I can change it and get `author` from store.